### ISSUE TITLE
Replace use of deprecated WebView with WKWebView

### ISSUE THREAD

**Issue by dbrajkovic** [subethaedit/SubEthaEdit#37]

**Comment by monkeydom** [comment]
Yeah - this should be done, probably needs a good set of exploration to still get all the functionality, resource loading, etc. in that is happening now.

**Comment by dbrajkovic** [comment]
I've done a lot of work with WKWebview in iOS, I'll be happy take a crack at it.

**Comment by monkeydom** [comment]
@dbrajkovic we'd like to try a nice catalina aligned release. Do you think you'll have a chance to look at that in that timeframe?

**Comment by dbrajkovic** [comment]
@monkeydom  Yeah.  Let me look try this week

**Comment by monkeydom** [comment]
Any luck here?

**Comment by jncn** [comment]
Still blocked on the custom `WKURLSchemeHandler` since it doesn't support HTTP, unfortunately. Progress can be found here: https://github.com/jncn/SubEthaEdit/pull/1
